doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -459,13 +459,24 @@ For example::
 Add a File to a Dataset
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Add a file to an existing Dataset. Description and tags are optional::
+When adding a file to a dataset, you can optionally specify the following:
+
+*   A description of the file.
+*   Tags and categories for the file.
+*   Terms of use and access for the file.
+
+
+Add a file to an existing Dataset. Description, terms of use and tags are optional::
 
     POST http://$SERVER/api/datasets/$id/add?key=$apiKey
 
-A more detailed "add" example using curl::
+A more detailed "add" examples using curl::
+
+    curl -H "X-Dataverse-key:$API_TOKEN" -X POST -F 'file=@data.tsv' -F 'jsonData={"description":"My description.","categories":["Data"], "termsOfUseAndAccess":{"termsType":"LICENSE_BASED", "license":"Apache Software License 2.0"}}' "https://example.dataverse.edu/api/datasets/:persistentId/add?persistentId=$PERSISTENT_ID"
```

Comment:
I think we should mention that we will have the list of licenses somewhere, or maybe in the task where we create such a endpoint.

Reply:
Not going to update docs to sync with future (and undecided) state. This might be done in this task that will add license listing endpoint.